content/site-policy/github-terms/github-corporate-terms-of-service.md

```diff
@@ -325,7 +325,7 @@ Upon Customer’s request for Expert Services, GitHub will provide an SOW detail
 
 GitHub reserves the right, at its sole discretion, to amend this Agreement at any time and will update this Agreement in the event of any such amendments. GitHub will notify Customer of material changes to this Agreement, such as price increases, at least 30 days prior to the change taking effect by posting a notice on the Service or sending email to the primary email address specified in your GitHub account. Customer's continued use of the Service after those 30 days constitutes agreement to those revisions of this Agreement. For any other modifications, Customer's continued use of the Service constitutes agreement to our revisions of this Agreement. Customer can view all changes to this Agreement in our [Site Policy](https://github.com/github/site-policy) repository.
 
-GitHub changes the Service via Updates and addition of new features. Nothwithstanding the foregoing, GitHub reserves the right at any time to modify or discontinue, temporarily or permanently, the Service (or any part of it) with or without notice.
+GitHub changes the Service via Updates and addition of new features. Notwithstanding the foregoing, GitHub reserves the right at any time to modify or discontinue, temporarily or permanently, the Service (or any part of it) with or without notice.
```

Comment:
Notably this is a site-policy, I'd expect a lawyer or someone to want to review...

#### Before
https://docs.github.com/en/site-policy/github-terms/github-corporate-terms-of-service#r-changes-to-the-service-or-terms
<img width="680" alt="image" src="https://github.com/github/docs/assets/2119212/8baf3cc8-a96b-414f-9149-74f9844fca8f">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/site-policy/github-terms/github-corporate-terms-of-service#r-changes-to-the-service-or-terms
<img width="676" alt="image" src="https://github.com/github/docs/assets/2119212/ff520673-e13e-48a5-85fb-38595394a64d">